packages/karma-puppeteer-launcher/src/index.cjs

```diff
@@ -50,6 +50,9 @@ function puppeteerBrowser(baseBrowserDecorator, config) {
         // See http://crbug.com/715363
         // We use this flag to work-around this issue.
         '--disable-dev-shm-usage',
+        // Try to prevent "Blocked attempt to create a WebMediaPlayer" warnings in Chrome.
+        // See crbug.com/1144736#c27
+        '--max-web-media-player-count=10000'
```

Comment:
Can we make this number higher?

Reply:
I don't think we have more than even 100 video players to be honest.

This number is already extremely high.

Reply:
it's just the 12 video media elements in the media library panel. They just stick around after we restore the fixture. If you pay close attention the the errors while you're watching a running test, the `Blocked attempt to create a WebMediaPlayer` will increment by 12 every time the fixture remounts.

Trying to look at memory profiles, but they're super hard to track. Could possibly be a react/chrome error like mentioned above